web/index.php

```diff
@@ -8,7 +8,7 @@
 
 include __DIR__ . '/../bootstrap.php';
 
-define('IS_PRODUCTION', Config::get('is_prod') === "yes");
+define('IS_PRODUCTION', Config::get(Config::IS_PROD));
```

Comment:
Let's make this `=== true` in case other local setups still have values that are truthy (coerce to true via `==`).